src/components/UI/Navigation/OrganizationDropdown.js

```diff
@@ -145,6 +147,18 @@ const OrganizationDropdown = ({
       .sort((a, b) => a.localeCompare(b));
   }, [organizations.items, organizations.isFetching]);
 
+  let triggerLabel = 'No Organizations';
+
+  if (sortedOrganizations.length !== 0) {
+    if (selectedOrganization.length > TriggerLabelMaxLength) {
+      triggerLabel = (
+        <>{selectedOrganization.slice(0, TriggerLabelMaxLength)}&hellip;</>
+      );
+    } else {
+      triggerLabel = selectedOrganization;
+    }
+  }
```

Comment:
Tried to do this with css overflow and ellipsis but couldn't make it working. The component uses many inline divs. This was quick and easy

Reply:
We actually have an element made specifically for this
https://github.com/giantswarm/happa/blob/a939dc81043fab10d0ee676f9cfc4927fd25c355/src/components/UI/Truncated.tsx

Reply:
:facepalm: I was searchting for this.. I tried to look for overflow or ellipsis :sweat_smile: